src/Nethermind/Nethermind.Core/IKeyValueStoreWithBatching.cs

```diff
@@ -19,10 +19,8 @@ namespace Nethermind.Core
 {
     public interface IKeyValueStoreWithBatching : IKeyValueStore
     {
-        byte[]? this[byte[] key] { get; set; }
-        
```

Comment:
Unused code deletion.  Is that a desired thing?  No implementation of this actually used this indexer, so removing it seemed reasonable (and was suggested by the compiler).  Maybe this had an indexer so plugins could use it?

Reply:
have you checked in all solutions? NDM, EthereumTest?

Reply:
Everything compiles and tests pass after this change, so I believe it isn't used.  It *could* be used by a third party plugin though.